eth-reference-tests/src/referenceTest/java/tech/pegasys/teku/reference/phase0/sanity/SanityBlocksTestExecutor.java

```diff
@@ -37,6 +38,8 @@
 public class SanityBlocksTestExecutor implements TestExecutor {
 
   private static final String EXPECTED_STATE_FILENAME = "post.ssz_snappy";
+  private static final String STATE_ROOT_MISMATCH_ERROR_MESSAGE =
+      "Block state root does NOT match the calculated " + "state root";
```

Comment:
should probably put these together given they didn't wrap...

Reply:
Argh xD